src/studio/nuxeo-studio/nuxeo-studio-release-notes/2018-03-05-studio-73-8.md

```diff
@@ -0,0 +1,97 @@
+---
+title: 'March 5, 2018'
+description: .
+tree_item_index: 990
+review:
+  comment: ''
+  date: '2018-03-05'
+  status: ok
+toc: true
+---
+
+{{! multiexcerpt name='summary-v73-8'}}
+Introducing:
+- A REST API to trigger Studio releases
+- A revamped editor for page providers results layout configuration
+- Ability to declare constraints for workflow variables and node variables
+{{! /multiexcerpt}}
+
+## REST API for Studio Releases
+
+A REST API to create releases for your Studio project is now available. This new feature comes bundled with a set of changes to make your development flow and continuous integration setup easier.
+
+### Simplified Maven GAV
+
+In any branch, Studio will now display a simplified Maven GAV for your snapshots, that you can use to set up your continuous integration.
+
+![]({{file name='studio-maven-gav-ci.png'}} ?border=true,w=600)
+
+Comparatively to previous Studio versions, this Maven GAV remains the same even after a release. Notice also that two options are provided depending on whether you want to test your work in progress, or only changes committed in the shared branch.
+
+### Simplified Release Creation UI
+
+The release creation popup is simplified in the Studio UI.
+
+![]({{file name='studio-release-ui.png'}} ?border=true,w=600)
+
+It does not make use anymore of a next version number. Instead, it offers suggestions based on the latest release performed in the current branch.
+
+### Rest API for Studio Releases
+
+Releases can also be performed through a REST API to automate your release process. Below is an example.
+
+```
+Request:
+
+POST https://connect.nuxeo.com/nuxeo/site/studio/v2/project/{project_id}/releases
+Authorization: Basic base64({username}:{password})
+Accept: application/json
+Content-Type: application/json
+
+{
+    "revision": "{branch_name}",
+    "versionName": "(M.m.p-q|MAJOR|MINOR|PATCH)"
+}
+
+Response:
+
+Content-Type: application/json
+
+{
+    "version": "M.m.p-q",
+    "bundleSymbolicName": "studio.extensions.{project_id}",
+    "mavenCoordinates": "{project_group_id}:{project_id}:M.m.p-q"
+}
+```
+
+Similarly to the Studio UI, notice that Studio can decide of the version number to create based on your input and last release performed. Check the [how to release your Studio project documentation]({{page page='how-to-tag-or-release-your-nuxeo-studio-project'}}) for more information on this new endpoint.
```

Comment:
Similarly to the Studio UI, Studio decides on the version number to create based on your input and last release performed.